tests/unit/Helpers/InstallerQuestionsTest.php

```diff
@@ -81,7 +89,8 @@ protected function setUp(): void {
     $this->projectDirectory = getcwd();
     $this->rootDirectory = $this->projectDirectory;
     $container = $this->createMock('Symfony\Component\DependencyInjection\ContainerInterface');
-    $this->acquiaCli = new Cli($this->projectDirectory, $this->rootDirectory, $output, $container);
+    $this->starterKitValidation = new StarterKitValidation($output);
```

Comment:
In the StarterKitValidation() method, make sure to review whether the method utilizes any output parameters. If it doesn't, you should then assess the situation and take appropriate action based on this validation.
```suggestion
    $this->starterKitValidation = new StarterKitValidation();
```

Reply:
done